app.py

```diff
@@ -74,6 +80,10 @@ def add_scores(value):
     # connect the metrics stream to publish data
     metrics_observable.connect()
 
+    if os.getenv("FLT_LIVE_METRIC_COLLECT","False") == "True":
+        sleep(60*20) # Run live metric collection and analysis for 20 minutes
```

Comment:
I guess this to keep the collector running. But why only for 20 minutes? Let's just keep it running forever.

Reply:
If I keep a while (1) loop here, then I don't know how to end the program gracefully if its running on openshift